tools/EMLassemblyline/.shed.yml

```diff
@@ -0,0 +1,44 @@
+categories:
+    - Ecology
+description: Tools to operate machine actionnable jobs to EML metadata
+long_description: |
+  Tools to operate machine actionnable jobs to EML metadata from MetaShRIMPS
+name: metashrimps 
+owner: ecology 
+remote_repository_url: https://github.com/tanguygen/metacure
+type: unrestricted
+
+repositories:

```

Comment:
please remove this and adopt this shed.yml to a suite based yml file: https://github.com/galaxyproject/tools-iuc/blob/acc495bd77aa3defdda7ec8d8b84a37d6832f2ee/tools/dram/.shed.yml#L17

Reply:
Done! THANK YOU!!!